guides/Step By Step Guide/TableOfContents.md

```diff
@@ -0,0 +1,56 @@
+Welcome to the Klasa Step by Step Guide! This guide is intended specifically for beginner developers in order to take you from learning about Klasa to being able to use all the other pages on this website with ease. The following is a list of things that this guide will cover:
+
+1. {@tutorial CreatingTheBot Understanding Klasa And Creating A Bot}
+	- What is Klasa?
```

Comment:
these all need links

Reply:
Could you please show me how to do one of these links. Then I can do it for all of them. How do i change the page and scroll to the right place for the user.

Reply:
```suggestion
	- {@scrollto What is Klasa?}
```

Although, `@scrollto` is only used when the table of contents is within the same`.md` file
So it might be better off just using `{@tutorial}` tags for the indexed list entries.

Reply:
I think someone mentioned that tutorial does not scroll on that page. It just goes to the top of that page.